ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/logging.j2

```diff
@@ -15,6 +15,30 @@
 {%     if logging.trap is defined and logging.trap is not none %}
 | Trap | {{ logging.trap }} |
 {%     endif %}
+{%     if logging.format is defined and logging.format is not none %}
+
+| Format Type | Setting |
+| ----------- | ------- |
+{%        if logging.format.timestamp is defined and logging.format.timestamp == 'high-resolution' %}
+| Timestamp | high-resolution |
+{%        else %}
+| Timestamp | traditional |
+{%        endif %}
+{%        if logging.format.hostname is defined and logging.format.hostname is not none %}
+{%              if logging.format.hostname is defined and logging.format.hostname == 'fqdn' %}
```

Comment:
Can we do something like:

```
{%              if logging.format.hostname is defined %}
| Hostname | {{logging.format.hostname}} |
{%              endif %}
```

With this approach, it can easily scale if we need to add more options: no need to update this table